cvat/settings/base.py

```diff
@@ -571,3 +575,47 @@ def add_ssh_keys():
     'SCHEMA_PATH_PREFIX': '/api',
     'SCHEMA_PATH_PREFIX_TRIM': False,
 }
+
+# allauth configuration
+USE_ALLAUTH_SOCIAL_ACCOUNTS = strtobool(os.getenv('USE_ALLAUTH_SOCIAL_ACCOUNTS') or 'False')
```

Comment:
`or 'False'` is really need here?

Reply:
if an env variable is set to empty string then strtobool will not be able to convert it to bool